src/client/containers/QuizPage/QuizPage.container.js

```diff
@@ -1,75 +1,82 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import PageHeader from '../../components/PageHeader/PageHeader.component';
 import QuestionComponent from '../../components/Question/Question.component';
 import QuizAnswers from '../../components/QuizAnswers/QuizAnswers.component';
 import SideMenu from '../../components/SideMenu/SideMenu.component';
 import Buttons from '../../components/Buttons/Buttons.component';
 import ProgressBar from '../../components/ProgressBar/ProgressBar.component';
-import { useHistory } from 'react-router-dom';
-import image1 from '../../assets/images/questionBackgrounds/question1background.png';
-import image2 from '../../assets/images/questionBackgrounds/question2background.png';
-import image3 from '../../assets/images/questionBackgrounds/question3background.png';
-import image4 from '../../assets/images/questionBackgrounds/question4background.png';
-import image5 from '../../assets/images/questionBackgrounds/question5background.png';
+import {  Link} from 'react-router-dom';
```

Comment:
Are you using prettier? 

Why do you have 2 spaces there? :)

Reply:
Yes, I'm using Prettier. It wasn't fixed because the database column names are not in camel case, further down the file. I have manually fixed the spacing.